rechnung/transactions.py

```diff
@@ -1,14 +1,7 @@
-import datetime
```

Comment:
This whole module should be removed in favor of your new bank statement module.

Reply:
agreed. But as I lack test data, I would prefer that you do it, or submit test data. :)